App_Use_Cases.tex

```diff
@@ -502,7 +508,85 @@ \subsection{Use Case Details}
 
 \littleheader{Related Constants}
 
-\refconst{PMIX_SUCCESS}
+\refconst{PMIX_SUCCESS}\\
 \refconst{PMIX_ERR_NOT_SUPPORTED}
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Cross-Version Compatability}
+\label{app:uc-cross-version}
+
+\subsection{Use Case Summary}
+
+The \ac{PMIx} interface serves as a conduit between clients (e.g., \ac{MPI} libraries), tools (e.g., debuggers), and servers (e.g., \acp{RM}).
+As such, it is probable that a process operating in one of these roles (e.g., as a client or tool) is running a different version of the same \ac{PMIx} implementation than the process with which it is communicating that is operating in a different role (e.g., as a server).
+For processes running in containers cross-version compatibility is especially important because the container image and the system software levels will naturally evolve and drift apart.
+As such, there is a need for the \ac{PMIx} implementation to provide cross-version compatibility.
+
+The responsibility for providing cross-version compatibility is a feature of the \ac{PMIx} implementation and not necessarily of the \ac{PMIx} standard.
+The \ac{PMIx} standard must strive to enable, and never limit, both the cross-version compatibility in the \ac{PMIx} implementation, and the ability for a \ac{PMIx} consumer to adapt to cross-version differences in capabilities.
+
+This use case is focused on cross-version compatibility between versions of the same \ac{PMIx} implementation and not between different \ac{PMIx} implementations.
```

Comment:
Subjective nit (feel free to take or leave):
```suggestion
This use case is focused on cross-version compatibility between different versions of the same \ac{PMIx} implementation and not between different \ac{PMIx} implementations.
```

Reply:
I pushed a new commit with this change.